libs/core/kiln_ai/adapters/model_adapters/test_base_adapter.py

```
@@ -97,28 +97,28 @@ async def test_model_provider_loads_and_caches(adapter, mock_provider):
         mock_loader.assert_not_called()
 
 
-async def test_model_provider_missing_names(base_task):
+async def test_model_provider_invalid_provider_model_name(base_task):
+    """Test error when model or provider name is missing"""
+    # Test with missing model name
+    with pytest.raises(ValueError, match="Input should be"):
+        adapter = MockAdapter(
```

Ruff (0.11.9)

Local variable `adapter` is assigned to but never used

Remove assignment to unused variable `adapter`

(F841)